### PR TITLE
Check for real height and width before adding to list

### PR DESCRIPTION
**Pull request by cicloon**
Fix the reservations map.

### CODE REVIEW

File: www/googlemaps-cdv-plugin.js [hunk 1]
```diff
@@ -830,7 +830,8 @@ App.prototype.setDiv = function(div) {
     }
 
     if (isDom(div)) {
-        var children = getAllChildren(div);;
+        var children = getAllChildren(div);
+        var newChildren = newGetAllChildren(div);
```
Comment (ciscou) [inline www/googlemaps-cdv-plugin.js]: this line shouldn't be here, should it?

Comment (cicloon) [inline www/googlemaps-cdv-plugin.js]: Right

**Comment by jmanuelrosa** [comment]
👍 😄

**Comment by ciscou** [comment]
👏

**Pull request merged**